src/Microsoft.ML.TimeSeries/STL/Loess.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// Loess is short for Robust Locally Weighted Regression and Smoothing Scatterplots.
+    /// </summary>
+    public class Loess
```

Comment:
Please, change the 3 classes defined on this file to be `internal` for the reasons explained here:
https://github.com/dotnet/machinelearning/pull/5202/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.csv#r440495855

Reply:
updated.

---
In reply to: [440508172](https://github.com/dotnet/machinelearning/pull/5202#discussion_r440508172) [](ancestors = 440508172)